feature/#710 added relnote for submission entity (#743)
Avaiga/taipy-doc: docs/relnotes.md

    `SQLTableDataNode`, and `MongoCollectionDataNode`.<br/>
    For more information, refer to [Append new data to a data node](./manuals/core/entities/data-node-mgt.md#append-new-data-to-a-data-node).

> - When submitting a `Scenario^`, a `Sequence^` or a `Task^`, we will create a `Submission^` entity internally, which will hold all of the
>     metadata related to the submission such as its overall status, its submission date, the ids of the `Job^` involved, etc. The latest
>     `Submission^` of a `Scenario^`, a `Sequence^` or a `Task^` can be retrieved by calling the new `taipy.get_latest_submission()^` method.

Reviewer: ```suggestion
- When submitting a `Scenario^`, a `Sequence^` or a `Task^`, Taipy automatically creates a
`Submission^` entity. It holds the metadata related to the submission such as its overall
submission status, its submission date, the ids of the `Job^` involved, etc. 
The latest `Submission^` of a `Scenario^`, a `Sequence^`, or a `Task^` can be retrieved by calling the new `taipy.get_latest_submission()^` method.
```

    `SQLTableDataNode`, and `MongoCollectionDataNode`.<br/>
    For more information, refer to [Append new data to a data node](./manuals/core/entities/data-node-mgt.md#append-new-data-to-a-data-node).

> - When submitting a `Scenario^`, a `Sequence^` or a `Task^`, Taipy automatically creates a

Reviewer 2: ```suggestion
- A new class called `Submission^` was added. It holds the meta-data (such as its status or submission date) of all entities that are submitted: `Scenario^`, `Sequence^`, and `Task^`.</br>
The function `taipy.get_latest_submission()^` was also added to retrieve the last submitted entity.
```

Reviewer 2: Globally, my point is that the paragraph should not repeat what can be found in the Reference Manual.
